webpack/assets/javascripts/react_app/components/Layout/components/TaxonomySwitcher.js

```diff
@@ -13,38 +13,33 @@ const TaxonomySwitcher = ({
   currentLocation,
   organizations,
   locations,
-  taxonomiesBool,
   isLoading,
   onLocationClick,
   onOrgClick,
 }) => (
   <Nav navbar pullLeft className="navbar-iconic">
-    {taxonomiesBool.organizations && (
-      <TaxonomyDropdown
-        taxonomyType="Organization"
-        id="organization-dropdown"
-        currentTaxonomy={currentOrganization}
-        taxonomies={organizations}
-        changeTaxonomy={onOrgClick}
-        anyTaxonomyText={ANY_ORGANIZATION_TEXT}
-        manageTaxonomyText="Manage Organizations"
-        anyTaxonomyURL="/organizations/clear"
-        manageTaxonomyURL="/organizations"
-      />
-    )}
-    {taxonomiesBool.locations && (
-      <TaxonomyDropdown
-        taxonomyType="Location"
-        id="location-dropdown"
-        currentTaxonomy={currentLocation}
-        taxonomies={locations}
-        changeTaxonomy={onLocationClick}
-        anyTaxonomyText={ANY_LOCATION_TEXT}
-        manageTaxonomyText="Manage Locations"
-        anyTaxonomyURL="/locations/clear"
-        manageTaxonomyURL="/locations"
-      />
-    )}
+    <TaxonomyDropdown
+      taxonomyType="Organization"
+      id="organization-dropdown"
+      currentTaxonomy={currentOrganization}
+      taxonomies={organizations}
+      changeTaxonomy={onOrgClick}
+      anyTaxonomyText="Any Organization"
+      manageTaxonomyText="Manage Organizations"
```

Comment:
These should be translated, previously the first used `ANY_ORGANIZATION_TEXT` but the second wasn't marked for translation. I'm fine with keeping the strings inline instead of using constants, but they should be translated.

Reply:
these strings are already translated here https://github.com/theforeman/foreman/blob/bcd650273e744537a40198c937184a04ff1a180b/webpack/assets/javascripts/react_app/components/Layout/components/TaxonomyDropdown.js#L63

![Screen Shot 2021-01-07 at 19 28 05](https://user-images.githubusercontent.com/11807069/103924780-329e4000-511f-11eb-839b-b4de8c4f768c.png)

doesn't `gettext` add these strings already?

Reply:
Looks like the Any ones are extracted for translation because of the constants in https://github.com/theforeman/foreman/blob/bcd650273e744537a40198c937184a04ff1a180b/webpack/assets/javascripts/react_app/components/Layout/LayoutConstants.js#L3-L4, (which don't make sense to translate there tbh), but the "Manage ..." strings aren't marked for extraction anywhere so they don't appear in the translation files at all, meaning there aren't translations for them in other languages. Also it's a bit odd that we have the string in one place and the call to __() in a different place?
In any case this isn't directly related to this PR, so I'll merge it and we can do another PR to fix the translations.